tools/pubsys/src/aws/publish_ami/mod.rs

```diff
@@ -22,6 +22,22 @@ use std::iter::FromIterator;
 use std::path::PathBuf;
 use structopt::{clap, StructOpt};
 
+#[derive(Debug, StructOpt)]
+pub(crate) struct ModifyOptions {
```

Comment:
I feel like `ModifyWho` be more intuitive. I'm struggling to think of what else would be under here that wouldn't simply be under `publish_args`.